apps/studio/components/grid/components/header/Header.tsx

```diff
@@ -420,7 +424,7 @@ const RowHeader = ({ table, sorts, filters }: RowHeaderProps) => {
               <Button
                 type="default"
                 size="tiny"
-                icon={<Trash />}
+                icon={<Trash className="text-destructive-600" />}
```

Comment:
hmm from a consistency POV: is this something we want to keep doing as a UI pattern?